iocore/eventsystem/CMakeLists.txt

```diff
@@ -49,7 +49,3 @@ target_link_libraries(inkevent
         tscpputil # transitive
         yaml-cpp::yaml-cpp # transitive
 )
-
-if(TS_USE_HWLOC)
-        target_link_libraries(inkevent PRIVATE hwloc::hwloc)
-endif()
```

Comment:
UnixEventProcessor.cc uses symbols from hwloc, so it is a usage requirement.

Reply:
I removed this because its pulled in transitively

Reply:
If the hwloc related code is that tightly coupled, it probably shouldn't be in 2 different components. If that's not the case, then imagine a scenario where we no longer need the hwloc code in tscore. Eventsystem will fail to compile, and the author will have to patch the inkevent CMakeLists as well, requiring a change in 2 places instead of 1.